std/stdio.d

```diff
@@ -3258,14 +3258,20 @@ struct LockingTextReader
 }
 
 /**
- * Indicates whether $(D T) is a file handle of some kind.
+ * Indicates whether $(D T) is a file handle, i.e. the type
+ * is implicitly convertable to $(LREF File) or a pointer to a
+ * $(REF FILE, core,stdc,stdio).
+ *
+ * Returns:
+ *      `bool`
```

Comment:
@WalterBright I'm pretty sure you have had the opposite position in the past.

Having a Returns section that repeats exactly what is in the main description is wasteful and somewhat useless.

> automated tooling

The only automated tooling that needs this is documentation generation. And it's just there to repeat the string you give it. I can't see how automated tooling can find any consistent semantic meaning from Returns descriptions.